Servers/UI/OJS.Servers.Ui/Controllers/Api/SubmissionsController.cs

```diff
@@ -55,21 +55,41 @@ public async Task<IActionResult> GetForProfile()
             .MapCollection<SubmissionForProfileResponseModel>()
             .ToOkResult();
 
+    /// <summary>
+    /// Gets a subset of submissions by specific problem and user and given take count.
+    /// </summary>
+    /// <param name="problemId">The id of the problem</param>
+    /// /// <param name="userId">The id of the user that we want to get the submissions for</param>
+    /// <param name="isOfficial">Should the submissions be only from compete mode</param>
```

Comment:
Written like that, it implies that if `isOfficial=true` we should get submissions from all modes.

Also, "mode" here is incorrect. This is "participation type" and we should be concise in docs.

Better description is "Return from `compete` or `practice` participation types"

Reply:
Corrected.